pkg/defaults/defaults.go

```diff
@@ -123,7 +123,7 @@ const (
 	DefaultAdamLogLevel = "warning" //min level of logs sent from EVE to Adam
 
 	DefaultQemuAccelDarwin     = "-machine q35,accel=hvf -cpu kvm64,kvmclock=off "
-	DefaultQemuAccelLinuxAmd64 = "-machine q35,accel=kvm,dump-guest-core=off,kernel-irqchip=split -cpu host,invtsc=on,kvmclock=off -device intel-iommu,intremap=on,caching-mode=on,aw-bits=48 "
+	DefaultQemuAccelLinuxAmd64 = "-machine q35,accel=kvm,dump-guest-core=off,kernel-irqchip=split -cpu host,-vmx-true-ctls,-vmx-secondary-ctls,-invtsc=on,kvmclock=off -device intel-iommu,intremap=on,caching-mode=on,aw-bits=48 "
```

Comment:
Looks like you should remove `-` before `invtsc=on`.

Reply:
Thanks @giggsoff, it was unintentionally, editing error. It's fixed.